tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -393,10 +415,7 @@ public async Task TestVulnerabilityStack(string name, string url)
                     span.Tags["_dd.stack"] = json;
                 }
 
-                foreach (var key in span.MetaStruct.Keys.ToArray())
-                {
-                    span.MetaStruct[key] = [];
-                }
+                span.MetaStruct.Remove("_dd.stack");
```

Comment:
I'd restore it to how it was. We don't want the content of the metastruct dict, but knowing what keys came can be useful. And we treat all the keys in the same way.